src/legacy/core_plugins/kbn_vislib_vis_types/public/pie.js

```diff
@@ -48,32 +49,7 @@ export default function HistogramVisType() {
     },
     editorConfig: {
       collections: {
-        legendPositions: [
-          {
-            text: i18n.translate('kbnVislibVisTypes.pie.editorConfig.legendPositions.leftText', {
-              defaultMessage: 'Left'
-            }),
-            value: 'left'
-          },
-          {
-            text: i18n.translate('kbnVislibVisTypes.pie.editorConfig.legendPositions.rightText', {
-              defaultMessage: 'Right'
-            }),
-            value: 'right'
-          },
-          {
-            text: i18n.translate('kbnVislibVisTypes.pie.editorConfig.legendPositions.topText', {
-              defaultMessage: 'Top'
-            }),
-            value: 'top'
-          },
-          {
-            text: i18n.translate('kbnVislibVisTypes.pie.editorConfig.legendPositions.bottomText', {
-              defaultMessage: 'Bottom'
-            }),
-            value: 'bottom'
-          },
-        ],
+        legendPositions: getPositions()
       },
       optionsTemplate: PieOptions,
       schemas: new Schemas([
```

Comment:
Let's use [AggGroupNames](https://github.com/elastic/kibana/blob/master/src/legacy/ui/public/vis/editors/default/agg_groups.ts#L22) enum for schema group name

Reply:
ok, done